test/conformance/ptp/ptp.go

```diff
@@ -1428,7 +1396,13 @@ func BasicClockSyncCheck(fullConfig testconfig.TestConfig, ptpConfig *ptpv1.PtpC
 	}
 
 	label, err := metrics.GetLabel(ptpConfig)
+	if err != nil {
+		logrus.Debugf("could not get label because of err: %s", err)
```

Comment:
Why not logrus.Errorf(), or, at least, logrus.Warnf()?

Reply:
it is ok to not have a label or a node but not both. The check is done later downstream in getClockIDForeign